pika/channel.py

```diff
@@ -206,8 +208,9 @@ def basic_consume(self, consumer_callback,
         self._validate_channel_and_callback(consumer_callback)
 
         # If a consumer tag was not passed, create one
-        consumer_tag = consumer_tag or 'ctag%i.%s' % (self.channel_number,
-                                                      uuid.uuid4().get_hex())
```

Comment:
`get_hex()` is the internal method for `hex` property. It was removed in Python 3